### ISSUE TITLE
Painting priority updates

### ISSUE THREAD

**Issue by HughJeffner** [HughJeffner/FFRK-LabMem#66]
~~Possible enemy priority list to replace the block list, (Behemoth dies quick)~~
~~Treasure vault could be better than a radiant painting~~
~~Avoidance options for master painting~~

**Comment by HughJeffner** [comment]
Added ability to set priority of radiant paintings

**Comment by HughJeffner** [comment]
List of enemies that may be prioritized by default (WIP)

- Behemoth
- Nidhogg

**Comment by alextcl** [comment]
A suggestion for handling painting priority:

Scale the painting priority to 10x , e.g. priority 1 become 10, and 9 become 90.

Example: farm 

Scenario: Avoid exploration if treasure behind
Bumping the exploration priority 20 down but before portal.
Assumption that Portal is priority 5 in selection (lookup portal value), take its 50 - 1 = 49 

Scenario: Avoid portal and master if treasure/exploration, still remaining painting behind
Bumping the portal/master priority down.
Take the lowest priority 90 + 1 = 91

Scenario: Prioritise and de-prioritise combat
Lookup the combat min and max priority
Take the highest combat priority = 60
Take the lowest combat priority = 80
  
If prioritise combat , then 60-1 = 59
otherwise to de-prioritise , 80+1 = 81

Scenario: shimmering painting priority below treasure
Assuming treasure priority is 10, then shimmering painting would be 11 

A setting JSON config structure as 
```json
"EnemyMap": 
{
    "Behemoth": {  
        "Priority": "High"/"Low", 
        "Description": "",
        "Enabled": true
    }
}
```

**Comment by HughJeffner** [comment]
Got the enemy priority list implemented in #132
